athena-federation-sdk-tools/pom.xml

```diff
@@ -70,6 +70,24 @@
             <version>${log4j2Version}</version>
             <scope>runtime</scope>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.amazonaws/aws-java-sdk-cloudformation -->
```

Comment:
This need to be test dependencies so they don't go into the jar that is fed to Lambda

Reply:
changed CloudFormation dependecy scope to provided `<scope>provided</scope>` in sdk-tools pom. Dependency will need to be provided in the specific connectors' pom with a `<scope>test</scope>`